deploy/olm-catalog/kogito-cloud-operator/0.4.0/app_v1alpha1_kogitoapp_crd.yaml

```diff
@@ -85,6 +85,45 @@ spec:
                     be compiled to run on native mode when Runtime is quarkus. See:
                     https://www.graalvm.org/docs/reference-manual/aot-compilation/'
                   type: boolean
+                resources:
+                  description: Resources for build pods. Default limits are 1GB RAM/0.5
+                    cpu on jvm and 2GB RAM/0.8 cpu for native builds.
```

Comment:
Can you please update the description to match current defaults for native build?